architectural-design-records/release-management-automation.md

```diff
@@ -0,0 +1,43 @@
+## Summary
+
+Use a Release-please GitHub action to automatically release new versions of our software on GitHub.
+
+### Problem
+
+Releasing new versions of software can be a time-consuming task, especially for teams that release software frequently. It can be easy to miss steps or make errors during the release process, and releasing software manually can take up valuable development time.
+
+### Decision
+
+We will use a Release-please GitHub action to automatically release new versions of our software on GitHub. The action will be triggered by a merge to main branch or specific commit tag, it will check all the CI checks pass, update the version of package, create a release and push it to the appropriate git tag and branch. Additionally, there'll be an action that will upload the app-of-apps.yaml to the release assets.
```

Comment:
- `...by a merge to main branch...` -> `by a merge to the main branch`
- `...it will check all the CI checks pass...` -> `it will check all the CI checks passed`
- `...update the version of package...` -> `update the version of the package`